src/folio-runner.js

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env node
+const yargs = require('yargs/yargs');
+const { spawn, execSync } = require('child_process');
+const _ = require('lodash');
+const path = require('path');
+const { shouldRecordVideo, getAbsolutePath, loadRunConfig, toHyphenated } = require('./utils');
+const { createjobLegacy, createJobShell } = require('./reporter');
+const SauceLabs = require('saucelabs').default;
+const { LOG_FILES } = require('./constants');
+const fs = require('fs');
+const glob = require('glob');
+
+const region = process.env.SAUCE_REGION || 'us-west-1';
+const jobName = process.env.SAUCE_JOB_NAME || `DevX Playwright Test Run - ${(new Date()).getTime()}`;
+
+const api = new SauceLabs({
+  user: process.env.SAUCE_USERNAME,
+  key: process.env.SAUCE_ACCESS_KEY,
+  region
+});
+
+async function run (nodeBin, runCfgPath, suiteName) {
+  runCfgPath = getAbsolutePath(runCfgPath);
+  const runCfg = await loadRunConfig(runCfgPath);
+  const cwd = path.dirname(runCfgPath);
+  let defaultArgs = {
+    param: {
+      headful: process.env.SAUCE_VM ? true : false,
+      video: shouldRecordVideo(),
+    },
+    reporter: 'junit,line',
+  };
+  let env = {
+    ...process.env,
+    FOLIO_JUNIT_OUTPUT_NAME: path.join(cwd, '__assets__', 'junit.xml')
+  };
+  let args = _.defaultsDeep(defaultArgs, {
+    output: path.join(cwd, '__assets__'),
+  });
+
+  const suite = _.find(runCfg.suites, ({name}) => name === suiteName);
+  if (!suite) {
+    throw new Error(`Could not find suite named '${suiteName}'`);
+  }
+  args = _.defaultsDeep(suite, args);
+
+  const folioBin = path.join(__dirname, '..', 'node_modules', '.bin', 'folio');
+  const procArgs = [folioBin];
+
+
+  for (let [key, value] of Object.entries(args)) {
+    key = toHyphenated(key);
+    if (key.toLowerCase() === 'name') {
+      continue;
+    }
+    if (value === '') {
+      continue;
+    }
+    if (key === 'param' || key === 'params') {
```

Comment:
@dpgraham what do you think about putting an example here:
`yml in` -> `command arg out`

like:
```
param:
  slowMo: true
```
is 
```
---param slowMo=true
```
Ad a link to https://github.com/microsoft/playwright-test/blob/master/README.md#run-the-test

Reply:
👍